test/Test/Gen.hs

```diff
@@ -1,9 +1,10 @@
 module Test.Gen where
 
+import Hedgehog
```

Comment:
All imports should have explicit import lists

Reply:
Yes, I need to fix this for some other modules as well